wwwroot/test/WMS/multiple_style_legend_url.xml

```diff
@@ -102,7 +102,7 @@
                         <Format>image/gif</Format>
                         <OnlineResource xmlns:xlink="http://www.w3.org/1999/xlink"
                                         xlink:type="simple"
-                                        xlink:href="http://www.example.com/legendUrl"/>
+                                        xlink:href="http://www.example.com/request=GetLegendGraphic?firstUrl"/>
```

Comment:
Our regex currently matches anything with the string `GetLegendGraphic`, but we might tighten it up in the future, in which case it'd be nice if this test still passed.